visualization.spectrum/resources/language/resource.language.de_de/strings.po

```diff
@@ -95,3 +95,23 @@ msgstr "%i°"
 msgctxt "#30019"
 msgid "Turn continuously"
 msgstr "Durchgehend drehen"
+
+msgctxt "#30020"
+msgid "Tilt"
+msgstr "Neigung"
+
+msgctxt "#30021"
+msgid "Base"
+msgstr "Basis"
+
+msgctxt "#30022"
+msgid "%+i%%"
+msgstr ""
+
+msgctxt "#30023"
+msgid "Field size"
+msgstr "Feldgröße"
+
+msgctxt "#30024"
+msgid "%i%%"
+msgstr ""
```

Comment:
As with use now the translation system by Weblate, would it better to leave de_de and the en_us unchanged.

Reply:
Done (reverted)